src/Game.js

```diff
@@ -1,5 +1,159 @@
 /* Manages the state of the game */
 class Game 
 {
+	constructor(board, players, endConditions)
+	{
+		this.board = board;
+		this.players = players;
+		this.endConditions = endConditions;
+		
+		/* Default turn order is alternating */
+		this.turnOrder = [
```

Comment:
Conceptually, I consider turn order to be a function of the last turn or the current board or the entire game for that matter. Effectively, that would mean this array would be less of an ordered list representing the sequence of turns and more of an unordered list of things that can have turns i.e. players. Each `Turn` then, upon completion, calls a callback function which checks whatever criteria it has and returns an index of the next turn to execute. This blurs the line between my conception, functional, and your conception, static.

Reply:
Make sure you read all the code in the PR. This case is specifically called out and addressed in the documentation for Turn.EndTurn(), which can override the default turn ordering. This array just sets that default, which right now is hardcoded to alternating, with any legal move. That will be configurable later, and part of that configuration will also have to get down into Turn.EndTurn(), or that processing will be moved elsewhere.